Packs/AnomaliSecurityAnalyticsAlerts/Integrations/AnomaliSecurityAnalyticsAlerts/AnomaliSecurityAnalyticsAlerts.py

```diff
@@ -0,0 +1,340 @@
+            human_readable = (
+                f"No results found for Job ID: {job_id}. "
+                f"Error message: {status_response.get('error')}. "
+                f"Please verify the Job ID and try again."
+            )
+            command_result = CommandResults(
+                outputs_prefix='ThreatstreamAlerts.SearchJobResults',
+                outputs={},
+                readable_output=human_readable,
+                raw_response=status_response
+            )
+            command_results.append(command_result)
+            continue
+
+        status_value = status_response.get('status')
+        if status_value is None or status_value.upper() != 'DONE':
+            human_readable = f"Job ID: {job_id} is still running. Current status: {status_value}."
+            command_result = CommandResults(
+                outputs_prefix='ThreatstreamAlerts.SearchJobResults',
+                outputs={"job_id": job_id, "status": status_value},
+                readable_output=human_readable,
+                raw_response=status_response
+            )
+            command_results.append(command_result)
+        else:
+            results_response = client.get_search_job_results(job_id, offset=offset, fetch_size=fetch_size)
+            if 'fields' in results_response and 'records' in results_response:
+                headers = results_response['fields']
+                records = results_response['records']
+                table_data = [dict(zip(headers, record)) for record in records]
+                human_readable = tableToMarkdown(name="Search Job Results",
+                                                 t=table_data,
+                                                 headers=headers,
+                                                 removeNull=True)
+            else:
+                human_readable = tableToMarkdown(name="Search Job Results",
+                                                 t=results_response,
+                                                 removeNull=True)
+            command_result = CommandResults(
+                outputs_prefix='ThreatstreamAlerts.SearchJobResults',
+                outputs_key_field='job_id',
+                outputs=results_response,
+                readable_output=human_readable,
+                raw_response=results_response
+            )
+            command_results.append(command_result)
+    return command_results
+
+
+def command_update_alert(client: Client, args: dict) -> CommandResults:
+    """Update the status or comment of an alert.
+
+    Args:
+        client (Client): Client object with request
+        args (dict): Usually demisto.args()
+
+    Returns:
+        CommandResults.
+    """
+    status = str(args.get('status'))
+    comment = str(args.get('comment'))
+    uuid_val = str(args.get('uuid'))
+    if not uuid_val:
+        raise DemistoException("Please provide 'uuid' parameter.")
+    if status == 'None' and comment == 'None':
```

Comment:
If we need either status or comment the condition should be with or 
```suggestion
    if status == 'None' or comment == 'None':
```